frontend/src/bundles/common/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+    Checkbox as CheckboxMUI,
+    FormControlLabel,
+    type SxProps,
+    type Theme,
+} from '@mui/material';
+
+type Properties = {
+    label?: string;
+    disabled?: boolean;
+    defaultChecked?: boolean;
+    checked?: boolean;
+    required?: boolean;
+    value?: string;
+    size?: 'small' | 'medium';
+    sx?: SxProps<Theme>;
```

Comment:
for styles we use className and modules

Reply:
Can`t we just extand MUI CheckboxProps or FormControlLabelProps interface to reuse in our custom component as a props interface?
In that way we will get rid of repetetive code and make component cleaner.
The only problem is that project require us to use types, not interfaces